src/include/miopen/conv/heur/heur.hpp

```diff
@@ -88,15 +88,15 @@ struct ConvHeur
         {
             cached = true;
             MIOPEN_LOG_I2("Cached heuristic result found");
-            std::vector<uint64_t> db_sol(db_res->size());
+            std::vector<std::pair<uint64_t, float>> db_sol(db_res->size());
             std::transform(db_res->begin(), db_res->end(), db_sol.begin(), [](boost::any id) {
-                return boost::any_cast<uint64_t>(id);
+                return boost::any_cast<std::pair<uint64_t, float>>(id);
             });
             if(miopen::IsLogging(LoggingLevel::Info2))
             {
                 std::stringstream ss;
                 for(auto& id : db_sol)
-                    ss << solver::Id{id}.ToString() << ", ";
+                    ss << solver::Id{id.first}.ToString() << ", ";
```

Comment:
It is worth to print also AI Heuristic Index here.

Reply:
Added printing of solver ID.